solidity/contracts/isms/ccip-read/AbstractCcipReadIsm.sol

```diff
@@ -21,10 +21,50 @@ import {PackageVersioned} from "../../PackageVersioned.sol";
  *    message. This functions selector should be provided as the `callbackFunction` payload
  *    for the OffchainLookup error
  */
-abstract contract AbstractCcipReadIsm is ICcipReadIsm, PackageVersioned {
+abstract contract AbstractCcipReadIsm is ICcipReadIsm, MailboxClient {
```

Comment:
Why is it a MailboxClient? Just for the Mailbox/process part? I dont think we are actually using it, so imo i'd rather just remove it